src/effector/__tests__/fork/serialize.test.ts

```diff
@@ -455,3 +455,70 @@ test('onlyChanges: false supported only in domain-based scopes', () => {
     serialize(scope, {onlyChanges: false})
   }).toThrowErrorMatchingInlineSnapshot(`"scope should be created from domain"`)
 })
+
+describe('serialize: missing sids', () => {
+  test('serialize: warns about missing sids', () => {
+    const error = jest.fn()
+    const consoleError = console.error
+    console.error = error
+
+    // forcing missing sid
+    // equals to situation, if user forgot to configure babel-plugin
+    // or did not install the sid manually
+    const $store = createStore('value', {sid: ''})
+
+    const scope = fork()
+
+    allSettled($store, {scope, params: 'scope value'})
+
+    const result = serialize(scope)
+    expect(result).toEqual({})
+    expect(scope.getState($store)).toEqual('scope value')
+    expect(error).toHaveBeenCalledWith(
+      'provided scope cannot be serialized in a reliable way because some stores are missing sid. Please, check that babel-plugin is working or provide sid manually',
+    )
+
+    console.error = consoleError
+  })
+  test('serialize: doesn not warn, if no sid is missing', () => {
+    const error = jest.fn()
+    const consoleError = console.error
+    console.error = error
+
+    const $store = createStore('value')
+
+    const scope = fork()
+
+    allSettled($store, {scope, params: 'scope value'})
+
+    const result = serialize(scope)
+    expect(result).toEqual({
+      [$store.sid as string]: 'scope value',
+    })
+    expect(scope.getState($store)).toEqual('scope value')
+    expect(error).toHaveBeenCalledTimes(0)
+    console.error = consoleError
```

Comment:
All interactions with environment should be performed in beforeEach/afterEach, otherwise when a single test will fail, it will broke console for rest of the suite

```ts
describe('serialize: missing sids', () => {
  let fn
  let consoleError
  beforeEach(() => {
    fn = jest.fn()
    consoleError = console.error
  })
  afterEach(() => {
    console.error = consoleError
  })
})
```